synapse/lib/storm.lark

```diff
@@ -314,7 +316,7 @@ PROPNAME: PROPS | UNIVNAME
 ABSPROP: PROPNAME
 ABSPROPNOUNIV: PROPS
 // A relative property
-relprop: RELNAME | ":" _varvalu
+relprop: ":" BASEPROP | ":" _varvalu
 
 // Similar to PROPS, but starts with a colon and allows :: as separator
 RELNAME: /\:\.?[a-z_][a-z0-9]*(?:(\:\:|\:|\.)[a-z0-9]+)*/
```

Comment:
this isn't used anymore. You're changing syntax here. You're disallowing `:.foo` now.